### ISSUE TITLE
What's the replacement of "Normal.passThrough()" in 1.0 version?

### ISSUE THREAD

**Issue by laphylee** [brookhong/Surfingkeys#1601]
Previously, I get into the passthrough mode when entering the fullscreen mode:

```js
document.onfullscreenchange = function (event) {
    console.log("FULL SCREEN CHANGE");
    if (document.fullscreenElement) {
        Normal.passThrough(); //enter to pass-through mode
    }
    else {
        Normal.passThrough(100); //back to normal mode after a very short time
    }
};
```

Since "Normal" is not visible in public API now, what's the replacement of "Normal.passThrough()"?

**Comment by rosshadden** [comment]
This is really smart. I'm going to use this once we know how to do it in the latest version.
I had a lot of mappings that used the `Normal` global and judging by the recently opened issues many others did too. It's kind of weird that `Visual` was moved to `api.Visual` but `Normal` is just entirely private.

**Comment by petobens** [comment]
@rosshadden is Visual working for you? I have the following mapping in my config
```js
mapkey('n', 'Next search result', function () {
    visual.next(false);
});
```
but it doesn't seem to work at all.. (I also tried with `Visual.next(false)`)

**Comment by brookhong** [comment]
`Normal.passThrough` published in 1.0.3
